website_sale_attribute_filter_category/static/src/less/website_sale_attribute_filter_category.less

```diff
@@ -0,0 +1,15 @@
+.attrib-category-radio {
+    margin: 0 20px;
+}
+
+[data-toggle="collapse"] .fas:before {
+  content: "\f105";
+}
+
+[aria-expanded="true"] .fas:before {
+  content: "\f107";
+}
+
+[data-toggle="collapse"].collapsed .fas:before {
+  content: "\f105";
```

Comment:
I'd appreciate some comments on this source code, because reading this isn't obvious what it does. I guess it's adding FA icons, but don't know which anyways.

Reply:
Done